creator-node/src/routes/tracks.js

```diff
@@ -351,6 +351,25 @@ module.exports = function (app) {
       return successResponse({ isDownloadable: true, cid: null })
     }
   }))
+
+  /** List all unlisted tracks for a user */
+  app.get('/tracks/unlisted', authMiddleware, handleResponse(async (req, res) => {
```

Comment:
fyi, the 'authMiddleware' is going to be changed in the notifs PR but i think the change applies here too - basically to require a Encoded-Data-Message and Encoded-Data-Signature. Adds an additional layer of security to prevent anyone from getting unlisted tracks via cnode. lets talk offline with @dmanjunath on what order to handle this in

Reply:
nvm